temporian/implementation/numpy/data/event.py

```diff
@@ -174,15 +174,14 @@ def from_dataframe(
 
         # check column dtypes, every dtype should be a key of DTYPE_MAPPING
         for column in df.columns:
-            # if dtype is object, convert to string
+            # if dtype is object, check if it is a string column
             if df[column].dtype.type is np.object_:
```

Comment:
What happens if the column is already `np.string_`? We should accept that too

Reply:
adding it to DTYPE_MAPPING like we discussed might be a solution, make sure to test to confirm